### ISSUE TITLE
Support --keep-context parameter on minikube stop

### ISSUE THREAD

**Issue by kschaab** [kubernetes/minikube#9001]
Please add the `--keep-context` parameter to `minikube stop` so that we can indicate to minikube to not switch contexts on stop.

**Comment by medyagh** [comment]
current behaviour:
```
medya@~/workspace/campwiz (master) $ kubectl config get-contexts
CURRENT   NAME       CLUSTER    AUTHINFO   NAMESPACE
          minikube   minikube   minikube   
*         p2         p2         p2         
          p3         p3         p3         

medya@~/workspace/campwiz (master) $ minikube stop -p p2
✋  Stopping node "p2"  ...
🛑  Powering off "p2" via SSH ...
🛑  1 nodes stopped.


medya@~/workspace/campwiz (master) $ kubectl config get-contexts
CURRENT   NAME       CLUSTER    AUTHINFO   NAMESPACE
          minikube   minikube   minikube   
          p2         p2         p2         
          p3         p3         p3   

```

in kubectl config get-contexts

but it does NOT have a * on it

because it is not Active

so you want us to Active the context, even though there is no cluster ?

I suspect that would add all sort of weird issues, such as you would get Api Errors, when you do, kubectl get pods

why would you want a Not Actibe cluster to be shown active?

if you want list of all context (including not active ones, you can get it form `kubectl config get-contexts`

Medya Ghazizadeh, Now
so I am not yet convinced if this is a feature that minikube should add, but I open to be convinced

**Comment by kschaab** [comment]
I agree, I'm not sure how useful this is.

**Comment by sharifelgamal** [comment]
This is fixed by #9044
